measures/2018/measures-data.json

```diff
@@ -44112,7 +44112,7 @@
     "nqfEMeasureId": null,
     "nqfId": null,
     "measureId": "441",
-    "description": "The IVD All-or-None Measure is one outcome measure (optimal control). The measure contains four goals. All four goals within a measure must be reached in order to meet that measure. The numerator for the all-or-none measure should be collected from the organization's total IVD denominator. All-or-None Outcome Measure (Optimal Control) - Using the IVD denominator optimal results include:\n- Most recent blood pressure (BP) measurement is less than 140/90 mm Hg -- And\n- Most recent tobacco status is Tobacco Free -- And\n- Daily Aspirin or Other Antiplatelet Unless Contraindicated -- And\n- Statin Use Unless Contraindicated",
+    "description": "The IVD All-or-None Measure is one outcome measure (optimal control). The measure contains four goals. All four goals within a measure must be reached in order to meet that measure. The numerator for the all-or-none measure should be collected from the organization's total IVD denominator. All-or-None Outcome Measure (Optimal Control) - Using the IVD denominator optimal results include:\r- Most recent blood pressure (BP) measurement is less than 140/90 mm Hg -- And\r- Most recent tobacco status is Tobacco Free -- And\r- Daily Aspirin or Other Antiplatelet Unless Contraindicated -- And\r- Statin Use Unless Contraindicated",
```

Comment:
Is there a reason this was changed from newlines to carriage returns? I see both in other places but it looks like newlines are a lot more common. Should we try to be consistent? Does it change anything switching them here?